docs/configure_sensors.md

```diff
@@ -0,0 +1,59 @@
+  - Restrictions:
+    - All sensors must have non-empty names.
+    - You cannot have two sensors with the same name in the database. Note that all sensor names are stored in lowercase in the database, so this restriction is NOT case sensitive.
+- Fields
+  - Once you decided on the sensor name, it's time to give your sensor some fields. For some sensors, such as a temperature sensor, one field is enough. In this case, it is perfectly acceptable to name your field the same as your sensor name.
+  - For other sensors, such as a triple-axis accelerometer, multiple fields are required. In this case, you can click the green "+" button at the bottom of the Action column to add a new field. 
+  - Notice that each field row also has an "X" in its Action column; if you accidentally inserted too many fields, you can always delete any of the fields with this "X" button. 
+    - However, there always must be at least one field per sensor. This is enforced by our UI, so don't worry about deleting too many fields; the app won't let you.
+  - For each field, there are three potential data types: numeric (float), character (string), and boolean. Choose whatever data type is appropriate for each individual field.
+  - Each field has a "Unit" option for user convenience. Units are not mandated.
+  - Restrictions
+    - A sensor cannot have two fields with the same name.
+    - All fields must have non-empty names.
+- Graph Type
+  - On the bottom of the add sensor form you will see a dropdown labeled Graph Type. Here you can choose the default way that data from the sensor will be viewed on Grafana. 
+  - Further graph customization options (bar graphs, absolute values, etc) are available within your Grafana instance.
+- Once you've added all of your sensor fields and chosen the default graph type for Grafana, hit the "Add New Sensor" button at the bottom of the form. The sensor will then be saved in the database and a panel with the graph type you chose will be created automatically in Grafana.
+
+### Adding GPS Sensor
+The GPS sensor receives special treatment in Mercury. In order to add a GPS Sensor, you must give exactly two fields with the names "latitude" and "longitude". You also must choose "Map" as your graph type. The sensor name can be whatever you wish, so long as it complies with the restrictions listed above. An example of a properly formatted GPS sensor can be seen below:
+![Addding GPS Sensor](imgs/add_sensor_gps.png)
+
+
+## Viewing Existing Sensors
+In order to view your existing sensors, navigate to the configure sensors page and scroll beneath the add sensor form. If you have any existing sensors, they will appear here, as seen below.
+![Existing Sensors](imgs/existing_sensors.png)
+
+Notice that each sensors has a unique, auto-generated ID number. This ID number is critical as it allows you to link the sensor you create through the UI with the actual physical sensor hardware. Sensor ID is not editable.
```

Comment:
@dandipietrantonio "Notice that each sensors" -> sensors should be singular, sensor